charts/minibroker/values.yaml

```diff
@@ -11,6 +11,13 @@ tls:
   # base-64 encoded PEM data for the private key matching the certificate
   key:
 
+# The service broker server configuration.
+broker:
+  # The deployment configuration.
+  deployment:
+    # A port for the service broker HTTP API.
+    port: 8080
```

Comment:
I don't understand why this wants to be configurable.  It's only used in the container; everybody outside the helm chart should be using the service (which is still on port 80).

Reply:
Indeed. My fault. Fixed with https://github.com/kubernetes-sigs/minibroker/pull/81/commits/e2dbcfd44fc89d859186db6608d199a3d28f67ec.